rococo-parachains/src/cli.rs

```diff
@@ -60,15 +60,15 @@ pub struct ExportGenesisStateCommand {
 	pub output: Option<PathBuf>,
 
 	/// Id of the parachain this state is for.
-	#[structopt(long, default_value = "100")]
+	#[structopt(long, default_value = "100", conflicts_with = "chain")]
```

Comment:
`parachain_id` has default value which means you can't pass `chain` arg because `parachain_id` always has value

Reply:
Does it complain at runtime?

Reply:
I don't know what your means, but you can simply verify what I said

```
jasl@localhost:~/Workspaces/BlockChain/cumulus on master$ target/release/rococo-collator export-genesis-state --chain tick
error: The argument '--chain <chain>' cannot be used with '--parachain-id <parachain-id>'

USAGE:
    rococo-collator export-genesis-state --chain <chain> --parachain-id <parachain-id>

For more information try --help
```

Reply:
Yeah that is at runtime. Hmmm. That is bad.